nexus-sdk/src/main/java/io/nexusrpc/handler/OperationStartDetails.java

```diff
@@ -0,0 +1,97 @@
+package io.nexusrpc.handler;
+
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Objects;
+import org.jspecify.annotations.Nullable;
+
+/** Details for handling operation start. */
+public class OperationStartDetails {
```

Comment:
Are we missing a `Headers` field like https://pkg.go.dev/github.com/nexus-rpc/sdk-go/nexus#StartOperationOptions?

Reply:
Looks like the headers made it into `Context` in Java. That's fine with me.

Reply:
:+1: Yeah it seemed very common across all calls